src/Aspire.Dashboard/SDK/Sdk.targets

```
@@ -0,0 +1,16 @@
+<Project>
+
+  <!-- *** BEGIN *** -->
+
+  <PropertyGroup>
+    <AspireDashboardDir Condition=" '$(AspireDashboardDir)' == '' ">$([MSBuild]::NormalizePath($(MSBuildThisFileDirectory), '..', 'tools'))</AspireDashboardDir>
```

nit: Use `NormalizeDirectory` as the property is a directory path, and this will ensure a trailing slash.
```suggestion
    <AspireDashboardDir Condition=" '$(AspireDashboardDir)' == '' ">$([MSBuild]::NormalizeDirectory($(MSBuildThisFileDirectory), '..', 'tools'))</AspireDashboardDir>
```

---

It would probably also be useful to add something like:
```xml
<AspireDashboardDir Condition="'$(AspireDashboardDir)' != ''">$([MSBuild]::EnsureTrailingSlash('$(AspireDashboardDir)'))</AspireDashboardDir>
```

This would cover the case where the property was overridden.

---

Good call, that'll avoid possible headaches when setting the properties manually.